pkg/config/file.go

```diff
@@ -89,6 +89,25 @@ http:
   # Make sure to use https:// if you are using TLS.
   public_url: "{{ .HTTP.PublicURL }}"
 
+  # The cross-origin request security options
+  cors:
+    # The allowed cross-origin headers
+    allowed_headers:
+       - "Accept"
+       - "Accept-Language"
+       - "Content-Language"
+       - "Origin"
+
+    # The allowed cross-origin URLs
+    allowed_origins:
+       - "" # do not allow any origins
```

Comment:
```suggestion
       #    - * # Allow all origins don't do this unless you know what you're doing ;)
```

Reply:
I agree with gorilla docs that returning * is unsafe and not required in any case
https://github.com/gorilla/handlers/blob/9c61bd81e701cf500437e1b516b675cdd3b73ca7/cors.go#L122

So I removed the suggestion of "*" to avoid giving users poor advice.

It might be better to offer a specific example here, like "https://website.com". Or not say anything at all.